easyaudit/utils.py

```diff
@@ -30,7 +32,12 @@ def get_field_value(obj, field):
             value = field.default if field.default is not NOT_PROVIDED else None
     else:
         try:
-            value = smart_str(getattr(obj, field.name, None))
+            for cls, resolver in resolver_map.items():
```

Comment:
This doesn't seem to allow customization for datetime fields (the other branch above). I'm also wondering if it's better call the resolver without `smart_str` when it's custom, to give us full control

Reply:
Shouldn't we still keep the timezone related stuff somehow? If we allow for datetime customization.